esp/platform/espcontext.cpp

```diff
@@ -503,10 +503,11 @@ class CEspContext : public CInterface, implements IEspContext
         return m_txSummary.get();
     }
 
-    virtual void addTraceSummaryValue(LogLevel logLevel, const char *name, const char *value)
+    // Update the interface of just this one method to show how it would be used
+    virtual void addTraceSummaryValue(LogLevel logLevel, const char *name, const char *value, TxSummaryStyle style = TxSummary_text)
```

Comment:
Can the style be set for the entire CTxSummary object rather than passed in for each value?  Or even better as a static member of CTxSummary and set once for the entire process?

Reply:
oh, I see now this is for which style it applies to, not how it will be output.